build.sh

```diff
@@ -75,6 +75,11 @@ function build_codemirror {
                text-hover \
     --local node_modules/uglify-js/bin/uglifyjs \
     > codemirror-compressed.js
+  exitcode=$?
+  if [ $exitcode -ne 0 ]; then
+    cat codemirror-compressed.js
```

Comment:
Nice catch.  Can you also delete `codemirror-compressed.js` in this case?  No point in leaving it around if it's not valid JavaScript anyway.